### PR TITLE
[#12] main page, header and common theme

### PR DESCRIPTION
**Pull request by EvelinSu**
added: the src folder, common theme, two types of header (full and default) 
changed: global styles, homepage

### CODE REVIEW

File: src/components/layout/header/HeaderStack.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+export const HeaderStack: FC<WithChild> = ({children}) => {
+    return (
+        <Stack
+            direction={"row"}
+            justifyContent={"space-between"}
+            alignItems={"center"}
+            width={1}
+            spacing={1}
+            px={3}
+            height={"60px"}
```
Comment (gitkoshelew) [inline src/components/layout/header/HeaderStack.tsx]: it looks like shouldn't be hardcoded

File: src/components/layout/header/HeaderStack.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                "& .MuiIconButton-root": {
+                    bgcolor: "background.default",
+                    p: 0.8
+                },
+                "& .MuiTab-root": {
+                    px: {xs: 0.5, md: 1},
+                    mx: {xs: 0.5, md: 2},
+                    minHeight: 25,
+                    textTransform: "initial",
+                    borderRadius: "10px",
```
Comment (gitkoshelew) [inline src/components/layout/header/HeaderStack.tsx]: also need transfer to theme

File: src/styles/theme/defaultTheme.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import {createTheme} from "@mui/material";
+
+export const defaultTheme = createTheme({
+    palette: {
+        mode: "light",
+        primary: {
+            main: "#2D3E6B",
+            contrastText: "#FFFFFF",
+            dark: "#243664",
```
Comment (gitkoshelew) [inline src/styles/theme/defaultTheme.ts]: dark and light are not better name choice. What if new theme will be vise-versa

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/styles/theme/defaultTheme.ts (hunk 3 above)
Comment (EvelinSu) [reply src/styles/theme/defaultTheme.ts]: these are the names of the Material UI theme, I just overwritten them with our colors https://mui.com/material-ui/customization/default-theme/

File: src/components/layout/header/HeaderStack.tsx (hunk 2 above)
Comment (EvelinSu) [reply src/components/layout/header/HeaderStack.tsx]: Ok

File: src/components/layout/header/HeaderStack.tsx (hunk 1 above)
Comment (EvelinSu) [reply src/components/layout/header/HeaderStack.tsx]: Ok, I remove the hardcoded height

**Review by gitkoshelew — DISMISSED** [review]

**Review by Zenater — DISMISSED** [review]

**Review by Zenater — APPROVED** [review]

**Review by visiteur85 — APPROVED** [review]

**Pull request merged**